NA-overview.rst

```diff
@@ -677,7 +677,7 @@ contributing code?
 If we go this route of writing up a decision process which includes
 such a dispute resolution mechanism, I think the meat of it should
 be a roadmap that potential contributers and developers can follow
-to gain influence over numpy. Numpy development needs broad support
+to gain influence over numpy. NumPy development needs broad support
```

Comment:
There's a `numpy` just before the period that should also be changed. And I have spotted a number of other occurrences of `numpy` that refer to the project, not the module, in this same file, e.g. line 650.